pkg/model/status.go

```diff
@@ -14,6 +14,14 @@
 
 package model
 
+// Fileshare status
+const (
+      FileShareCreating       = "in progress"
```

Comment:
Will these status be visible to end user in any case? like in the dashboard or in in future, WF? Then we should have good formatting. 'In progress', Available ...